packages/mongodb-memory-server-core/src/util/getos/index.ts

```diff
@@ -169,7 +174,7 @@ async function tryOSRelease(): Promise<LinuxOS | undefined> {
  */
 async function tryFirstReleaseFile(): Promise<LinuxOS | undefined> {
   try {
-    const file = (await fspromises.readdir('/etc')).filter(
+    const file = fs.readdirSync('/etc').filter(
```

Comment:
why change this to be sync again?

Reply:
good question. no real reason, perhaps a bit consistency. are there real differences between using the sync methods and a promise that you immediately await for?

Reply:
in this case not really, but in 7.0.0 i tried to make things less "sync-blocking"

Reply:
I read up a little and don't see any apparent downsides in using either of those (other than the taste-argument) in this place. I'd go with whatever you prefer. I you want me to revert, I'll do that.

Reply:
And I'm a bit confused. Wasn't that using the `promisify` util before? If so, perhaps I wanted to remove that not realizing that you already did 🎉

Reply:
like i said, in this case it dosnt really matter, but i tried to use less sync things wherever possible

Reply:
> And I'm a bit confused. Wasn't that using the `promisify` util before? If so, perhaps I wanted to remove that not realizing that you already did tada

yes, in 6.x it was using this, but since 7.0 (betas) i moved it to use https://nodejs.org/api/fs.html#fs_fs_promises_api

Reply:
will revert.